xstate-chatbot/nodejs/src/env-variables.js

```diff
@@ -59,6 +59,11 @@ const envVariables = {
     kafkaConsumerGroupId: process.env.KAFKA_CONSUMER_GROUP_ID || 'xstate-chatbot',
   },
 
+  egovServices: {
+    egovServicesHost: process.env.EGOV_SERVICES_HOST || 'https://dev.digit.org/',
+    egovFilestoreServiceDownloadEndpoint: process.env.EGOV_FILESTORE_SERVICE_DOWNLOAD_ENDPOINT || "filestore/v1/files/url",
```

Comment:
Copy the upload link too. It would be getting used in Value First Adapter

Reply:
added and committed